src/components/facet-line/Dropdown.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import clsx from "clsx";
+
+import iconTriangle from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-triangle.svg";
+import iconExpandMore from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";
+
+export type DropdownItem = {
+  title: string;
+  href?: string;
+  disabled?: boolean;
+  value?: unknown;
+  selected?: boolean;
+};
+
+export const DropdownIcon = ({
```

Comment:
Please refactor this dropdown element into a general reusable component.

It seems as if it is getting duplicated throughout the project. It would be preferable to have it separately like we have for checkboxes and date-input.

Reply:
I have made a reusable component ready but I made a separate PR to use it in the places we use select now
https://github.com/danskernesdigitalebibliotek/dpl-react/pull/225